Libraries/LibWeb/Loader/ResourceLoader.cpp

```diff
@@ -575,9 +575,14 @@ void ResourceLoader::handle_network_response_headers(LoadRequest const& request,
     if (!request.page())
         return;
 
-    for (auto const& [header, value] : response_headers.headers()) {
-        if (header.equals_ignoring_ascii_case("Set-Cookie"sv)) {
-            store_response_cookies(*request.page(), request.url().value(), value);
+    if (request.respects_set_cookie()) {
+        // From https://fetch.spec.whatwg.org/#http-network-fetch:
```

Comment:
nit: This should point to this url: `https://fetch.spec.whatwg.org/#concept-http-network-fetch`. Then it is easier to search through the codebase and find the corresponding method in `Fetch/Fetching/Fetching.cpp`.

Reply:
Wasn't aware of this, thanks! Updated.